test/targets.cpp

```diff
@@ -43,7 +43,7 @@ TEST_CASE(targets)
 {
     // GCC doesn't load libmigraphx_ref unless necesssary even though it is linked to the test.
     // Force it to load by making ref target
-#if defined(__GNUC__) && !defined(__clang__)
+#if defined(__GNUC__) && !defined(__clang__) || defined(_MSC_VER)
```

Comment:
We have moved all Ref tests under `test/ref` therefore we should be able to remove `migraphx_ref` from here. 

https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/5139b9308a7b8deda201e972cc24ffa6c78c9036/test/CMakeLists.txt#L28 

And after that, we can remove defines for clang, gcc or msc_ver and call to `make_target("ref")`. In that case, it would dynamically load `migraphx_ref` for all cases including clang, gcc and windows.

Reply:
Thank you for quick answer, I ran `test_ref` on Windows and it is working.

Reply:
Alright, I will commit new changes then. Thank you.